server/service/src/standard_reports.rs

```diff
@@ -0,0 +1,88 @@
+use repository::{
+    ContextType, FormSchemaJson, FormSchemaRowRepository, ReportRow, ReportRowRepository,
+    StorageConnection, StorageConnectionManager,
+};
+use rust_embed::RustEmbed;
+use thiserror::Error;
+
+use crate::report::definition::ReportDefinition;
+use log::info;
+use serde::{Deserialize, Serialize};
+
+#[derive(RustEmbed)]
+// Relative to server/Cargo.toml
+#[folder = "../reports/generated"]
+
+pub struct EmbeddedStandardReports;
+
+#[derive(Debug, Error)]
+#[error("No standard reports found")]
+pub struct StandardReportsError;
+
+#[derive(Clone)]
+pub struct StandardReports;
+
+impl StandardReports {
+    // Load embedded reports
+    pub fn load_reports(connection_manager: StorageConnectionManager) -> Result<(), anyhow::Error> {
```

Comment:
I think this can just take a connection? Then you can avoid cloning the connection manager?